extensions/http-curl/client/HTTPClient.cpp

```diff
@@ -437,13 +438,13 @@ void HTTPClient::setFollowRedirects(bool follow) {
 }
 
 bool HTTPClient::isValidHttpHeaderField(std::string_view field_name) {
-  if (field_name.size() == 0) {
+  if (field_name.empty()) {
     return false;
   }
 
   // RFC822 3.1.2: The  field-name must be composed of printable ASCII characters
   // (i.e., characters that  have  values  between  33.  and  126., decimal, except colon).
-  for (auto ch : field_name) {
+  for (auto ch : field_name) {  // NOLINT(readability-use-anyofallof)
```

Comment:
```suggestion
  return std::all_of(std::begin(field_name), std::end(field_name), [](char c) { return c >= 33 && c <= 126 && c != ':'; });
```

Reply:
IMO this is a rare case where I would keep the comment to reference the RFC number and to keep the actual quote from the RFC.

Reply:
Updated in 85e2a718a128ed49cdbac0e41d23628cf5a1a98c